content/docs/examples/advanced-gateways/http-proxy/index.md

```diff
@@ -201,11 +205,12 @@ In the following section you are going to configure traffic from Istio-enabled p
 ## Understanding what happened
 
 In this example you configured access to an external legacy HTTPS proxy from the Istio service mesh. To simulate a
-remote HTTPS proxy, you deployed an HTTPS proxy in a separate namespace, without Istio automatic sidecar injection. In addition, for simulation of a remote proxy, you addressed the HTTPS proxy by its IP address and not by a Kubernetes
-service.
+remote HTTPS proxy, you deployed an HTTPS proxy in a separate namespace, without Istio automatic sidecar injection.
+In addition, for simulation of a remote proxy, you addressed the HTTPS proxy by its IP address and not by the domain
+name of a Kubernetes service.
```

Comment:
I love the fact that you are summarizing what the readers did on the example. I would advise you to do so as a numbered list instead:

```suggestion
In this example, you took the following steps to configure access to an external legacy HTTPS proxy from an Istio service mesh:

1. Deployed an HTTPS proxy in a separate namespace without Istio automatic sidecar injection to simulate a remote HTTPS proxy.
2. Addressed the HTTPS proxy by its IP address and not by the domain name of a Kubernetes service to maintain the simulation of a remote proxy.
```
You would need to continue with the content of the next paragraph. The numbered steps provide additional clarity to readers. If you can, try to be more high-level with this overarching flow. You already covered the nit and grit of a lot of it above.